exercises/flatten-array/flatten-array_spec.lua

```diff
@@ -33,4 +33,11 @@ describe('flatten-array', function()
       flatten({ { { {  } } }, { { } } })
     )
   end)
+
+  it('should flatten array from readme example', function()
+    assert.are.same(
+      {1,2,3,4,5},
+      flatten({1,{2,3,nil,4},{null},5})
```

Comment:
This is something that we can work around by using a different value for `nil` (some singleton table that is guaranteed to be unique, for example). This will mean that we still have a sequence so `ipairs` and `#` will be valid, but we'll need to have a special case for "nil".